### DISCUSSION TITLE
question about zIndex @twilio-paste/design-tokens/dist/tokens.es6.js

### DISCUSSION THREAD

**Discussion by anavahonskaya** [twilio-labs/paste#1640] [General] [answered]
Hello,

I am completely new in this area. Our project is happy to use yours for our purpose. During migration from 6.0.1 to 10.0.0 version I explored one the files that is used in our project - @twilio-paste/design-tokens/dist/tokens.es6.js and I have two questions regarding it.
1. The code was "zipped", no separate lines are there any more. Was is done by purpose or you have an option to leave it reader-friendly in the new version? :) 
2. At the end of the file I found that 
```
export const zIndices = {
zIndex0,
zIndex10,
zIndex20,
zIndex30,
zIndex40,
zIndex50,
zIndex60,
zIndex70,
zIndex80,
zIndex90,
};
```
was changed to 
`export const zIndices={zIndex0:"0",zIndex10:"10",zIndex20:"20",zIndex30:"30",zIndex40:"40",zIndex50:"50",zIndex60:"60",zIndex70:"70",zIndex80:"80",zIndex90:"90"};
`
and somewhere in the middle of the long code line in the new version it is still exists:
`export const zIndex0 = "0";export const zIndex10="10";export const zIndex20="20";export const zIndex30="30";export const zIndex40="40";export const zIndex50="50";export const zIndex60="60";export const zIndex70="70";export const zIndex80="80";export const zIndex90="90";`

So, my question again - was it done on purpose or it's something unexpected?

I care particularly about this file because currently it is patched in our project. So, it order to have well-read easy-to-modify-or-remove patch, we need well-structured original file code. In this case we can easily find which line to compare. Otherwise patch looks like removing all the file code that is one line and adding another long line that is your file with a minor zIndex modifications.

Thank you very much for attention!

Best wishes,
Alena

**Comment by nkrantz** [comment, marked as answer]
Hi Alena,

Thanks for starting a Discussion! The answer to question 1 is that the minification was intentional. Regardless of the structure of the file, we maintain the API, so that structure hasn't changed.

We don't generally recommend patching the design tokens file in projects -- we recommend importing it and merging it with your work. 

It would help to have a little more context on what you're trying to achieve here so that we can give some better guidance.

Thanks!

**Reply by anavahonskaya** [reply]
Hi Nora,

Thank you very much for quick informative reply and offering your help! Indeed, patching dependency is not what we enjoy the most at work :) In our case we increase zIndexes. As I mentioned, I am completely new and I don't really know what it means to "import it and merge it with our work", so that I passed your reply to our team. Thank you very much!

Best wishes,
Alena

**Reply by gloriliale** [reply]
Hi Alena, I hope that this recommendation resolves the issue! 

We're happy to help if you need further support, just let us know and give us more specifics about the goal you're trying to achieve and your current process for it :)

**Reply by anavahonskaya** [reply]
Hello, 

I consulted with a team and have one more question to you :) Could you please clarify what Nora means under "we recommend importing it and merging it with your work"? As I already mentioned, we increased zIndexes using patch. Could you please guide us how it is possible to increase zIndexes in a more appropriate way? Maybe you have a link to the example of the approach you recommended?

Thank you very much!

Best wishes,
Alena

**Reply by SiTaggart** [reply]
Hi,

I guess my first question is in what context are you needing to change the zIndexes? 

What is it about the app you are working in that requires it? 

We don't really recommend you changing them at all when using the system as other parts of the system rely on them. So if you change them, you might introduce unexpected behaviors that we no longer have any control over.

First, we'd love to start exploring alternatives to changing the values at all, as much detail as possible about the app would be useful to get us started
